lib/Smokeping.pm

```diff
@@ -1500,7 +1500,7 @@ sub get_detail ($$$$;$){
             }
            $page .= "<div class=\"panel-body\">";
            $page .= qq|<IMG alt="" id="zoom" width="$xs{''}" height="$ys{''}" SRC="${imghref}_${end}_${start}.png">| ;
-           $page .= $q->start_form(-method=>'POST', -id=>'range_form')
+           $page .= $q->start_form(-method=>'POST', -id=>'range_form', -action=>$cfg->{General}{cgiurl})
```

Comment:
this should happen by default ... but can't hurt either :)

Reply:
for me it doesn't happen by default :/ which caused an 404 when pressing the "Generate" button